cassdegrees/templates/widgets/rules.html

```diff
@@ -58,6 +58,31 @@
     </fieldset>
 </script>
 
+<script type="text/x-template" id="subjectAreaSpecificRuleTemplate">
```

Comment:
```suggestion
<script type="text/x-template" id="subjectAreaRuleTemplate">
```

Remove use of 'specific' as described above